### DISCUSSION TITLE
Features I think would be cool!

### DISCUSSION THREAD

**Discussion by itsKayaVo** [keiwando/toonsquid#598] [Ideas]
Here are some features that multiple/many people could enjoy in ToonSquid!
- Adobe Animate Import and Export ability
-Built-in effects for animation layers or just the symbol layers in general (Automatable)
     - Gaussian Blur
     - Motion Blur
     - Chromatic Aberration
     - Glitch
     - Mosaic
     - Pixelate
- Procreate Pen Import Ability
-Better Vector Pen Options
-An option so the app remembers the **last** tweening option you use and uses it for the next time you make a keyframe besides loading linear every time
-Keyframe picker (Added option besides using markers to pick, not replacing markers, but adding another option beside it)
     - An added option for that should be like the Looping options in Adobe Animate "Loop, Play Once, Single Frame, Play once Backwards, Loop Backwards"
     - Color effects "Brightness, Tint, Alpha, Etc."
     - EXPORT AS SPRITESHEET, you can select multiple or just one - symbol, and an option appears for exporting as sprite sheets!
     - Exporting XMLs could be nice as well for the sprite sheets!
- The ability to **warp Vector** besides turning it into raster
-Skew Option for transforming raster and **vector**
-Bones or Ik and Fk Rigging
- An Option for Pens could be "Smooth after or during drawing"

Thats all I have! Thank you for reading :D

**Comment by Lioncub901** [comment]
Honestly I don’t think Procreate brush support would be possible since I don’t think the Procreate devs would just give support for their brush type to anyone. Digital art companies like to keep the brushes to themselves other than Photoshop (which is why it is industry standard). The other request a pretty cool though.
